src/main/webapp/resources/js/pages/projects/metadata/MetadataTemplatesList.jsx

```diff
@@ -100,7 +152,12 @@ export function MetadataTemplatesList({ projectId }) {
       }}
       dataSource={templates}
       renderItem={(item) => (
-        <List.Item className="t-m-template" actions={getActionsForItem(item)}>
+        <HoverItem
+          className="t-m-template"
+          actions={
+            item.identifier != ALL_FIELDS_TEMPLATE_ID && getActionsForItem(item)
```

Comment:
You check `item.identifier != ALL_FIELDS_TEMPLATE_ID` multiple times, create a constant for this.

Reply:
Do you think it would be worth rendering the renderItem return value as it's own component now, it is really big?

Reply:
Added const in 8e63a642f3f46bf750d3af026924c0835221c272. Don't really think it would be worth it as it's the only place we are listing the templates. If later on we decide to list in multiple places for whatever reason then we can move it into its own component